src/app/modules/item/pages/item-display/item-display.component.ts

```diff
@@ -117,7 +124,7 @@ export class ItemDisplayComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   updateHeight(): void {
-    this.task?.getHeight(this.setHeight.bind(this));
+    this.task?.getHeight().subscribe(this.setHeight.bind(this));
```

Comment:
More readable and the way we do in the project:
```suggestion
    this.task?.getHeight().subscribe(val => this.setHeight(val));
```

Reply:
Fixed.